packages/core/test/integration/support.ts

```diff
@@ -1,19 +1,17 @@
-import assert from 'node:assert';
 import pTimeout from 'p-timeout';
 import { QueryTypes } from '@sequelize/core';
 import type { AbstractQuery } from '@sequelize/core/_non-semver-use-at-your-own-risk_/dialects/abstract/query.js';
 import * as Support from '../support';
 
 // Mocha still relies on 'this' https://github.com/mochajs/mocha/issues/2657
```

Comment:
Do we still need this comment here now that the rule is not disabled?

Reply:
Removed